inference-engine/tests_deprecated/unit/engines/gna/i16_quantisation_test.cpp

```diff
@@ -160,34 +160,34 @@ TEST_F(I16QuantisationTest, OnlyAffineWithNanScaleFactorFails) {
         .propagate_forward().throws();
 }
 
-TEST_F(I16QuantisationTest, OnlyAffineWithInfScaleFactorFails) {
+TEST_F(I16QuantisationTest, DISABLED_OnlyAffineWithInfScaleFactorFails) {
     gna()
         .onInferModel(Fc2DOutputModel())
         .withInfScaleFactor()
         .propagate_forward().throws();
 }
 
-TEST_F(I16QuantisationTest, AffineToMemoryWillResultInActivationInsertion) {
+TEST_F(I16QuantisationTest, DISABLED_AffineToMemoryWillResultInActivationInsertion) {
     assert_that()
         .onInferModel(affineToMemoryModel())
         .inNotCompactMode().withGNAConfig(GNA_CONFIG_KEY(SCALE_FACTOR), 1.0f)
         .gna().propagate_forward().called_with().pwl_inserted_into_nnet();
 }
 
-TEST_F(I16QuantisationTest, EltwiseToMemoryWithNoOutputActivationInsertion) {
+TEST_F(I16QuantisationTest, DISABLED_EltwiseToMemoryWithNoOutputActivationInsertion) {
     assert_that().inNotCompactMode().withGNAConfig(GNA_CONFIG_KEY(SCALE_FACTOR), 1.0f)
     .onInferModel(eltwiseToMemoryModelNoOutput(), [](CNNNetwork & net){
             net.addOutput("Eltwise_8");
         }).gna().propagate_forward().called_with().pwl_inserted_into_nnet();
 }
 
-TEST_F(I16QuantisationTest, EltwiseToMemory_ActivationInsertion) {
+TEST_F(I16QuantisationTest, DISABLED_EltwiseToMemory_ActivationInsertion) {
     assert_that().onInferModel(eltwiseToMemoryModel()).withGNAConfig(GNA_CONFIG_KEY(SCALE_FACTOR), 1.0f)
         .inNotCompactMode().gna().propagate_forward().called_with().pwl_inserted_into_nnet();
 }
 
 
-TEST_F(I16QuantisationTest, SplitFollowedByActivation_DummyDiagonalAffineInsertion) {
```

Comment:
The tests should be modified to re-enable.
@sirzabek consider to create an issue (or add to some existing issue to not forget.)

Reply:
Added to existing issue 55749.